cparser/Elab.ml

```diff
@@ -1737,16 +1785,22 @@ let elab_expr vararg loc env a =
       let b1,env = elab env a1 in
       let b2,env = elab env a2 in
       if List.mem AConst (attributes_of_type env b1.etyp) then
-        err "left-hand side of assignment has 'const' type";
+        error "read-only variable is not assignable";
       if not (is_modifiable_lvalue env b1) then
-        err "left-hand side of assignment is not a modifiable l-value";
+        err "expression is not assignable";
```

Comment:
Debatable.  "modifiable lvalue" is ISO C speak, but I can agree that "assignable" is easier to understand.